align_your_own.ipynb

```diff
@@ -0,0 +1,206 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# So, show me how to align two vector spaces for myself!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "No problem. We're going to run through the example given in the README again, and show you how to learn your own transformation to align the French vector space to the Russian vector space.\n",
+    "\n",
+    "First, let's define a few simple functions..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from fasttext import FastVector\n",
+    "\n",
+    "# from https://stackoverflow.com/questions/21030391/how-to-normalize-array-numpy\n",
+    "def normalized(a, axis=-1, order=2):\n",
+    "    \"\"\"Utility function to normalize the rows of a numpy array.\"\"\"\n",
+    "    l2 = np.atleast_1d(np.linalg.norm(a, order, axis))\n",
+    "    l2[l2==0] = 1\n",
+    "    return a / np.expand_dims(l2, axis)\n",
+    "\n",
+    "def make_training_matrices(source_dictionary, target_dictionary, bilingual_dictionary):\n",
+    "    \"\"\"source and target dictionaries are the FastVector objects of source/target languages.\n",
+    "    bilingual_dictionary is a list of translation pair tuples [(source_word, target_word), ...].\"\"\"\n",
```

Comment:
Minor issue: This doc text is too wide to display nicely when looking at the file in github. Please adjust :)